btk_input.py

```diff
@@ -157,15 +157,17 @@ def get_catalog(user_config_dict, catalog_name, selection_function_name, verbose
     else:
         utils_filename = os.path.join(os.path.dirname(btk.__file__), "utils.py")
         selection_function = None
-    catalog = btk.catalog.load_catalog(
-        catalog_name, selection_function=selection_function
-    )
+    catalog = btk.catalog.WLDCatalog.from_file(catalog_name)
```

Comment:
Thanks for updating the btk_input file, I am not sure it actually worked before though, it will need some working over (later).

Reply:
Sure, it needs to be incorporated in the module for easier use.